pkgs/development/python-modules/ludios_wpull/default.nix

```
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, chardet
+, dnspython
+, html5-parser
+, lxml
+, namedlist
+, psutil
+, setuptools
+, sqlalchemy
+, tornado
+, yapsy
+, python
+, pythonOlder
+# Override Python packages using
+# self: super: { pkg = super.pkg.overridePythonAttrs (oldAttrs: { ... }); }
+# Applied after defaultOverrides
+, packageOverrides ? self: super: {}
+}:
+
+let
+  defaultOverrides = [
+    # override the version of packages pinned in pyproject.toml
+    (self: super: {
+      sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
+        version = "2.0.23";
+        src = oldAttrs.src.override {
+          inherit version;
+          rev = "refs/tags/rel_${lib.replaceStrings [ "." ] [ "_" ] version}";
+          hash = "sha256-UIQlYsiZk1fzxDxKHNTBL06Ua/xPZQ15H5coEdkWc90=";
+        };
+      });
+    })
+  ];
+  py = python.override {
+    # Put packageOverrides at the start so they are applied after defaultOverrides
+    packageOverrides = lib.foldr lib.composeExtensions (self: super: { }) ([ packageOverrides ] ++ defaultOverrides);
+  };
+in with py.pkgs; buildPythonPackage rec {
+  pname = "ludios_wpull";
+  version = "5.0.3";
+
+  disabled = pythonOlder "3.12";
+
+  format = "pyproject";
```

```suggestion
  pyproject = true;
```

this will use pypa to build package